README.md

```diff
@@ -22,6 +22,19 @@ config :movie_wager_api, MovieWagerApi.Repo,
 Similarly, you will want to follow the same steps for `test.secret.example.exs`.
 The secret files are not checked into source control, so don't worry about your credentials being shared.
 
+## Setting up Twitter Authentication
+
+The `dev.secret.exs` and `test.secret.exs` files also include information for the twitter OAuth API.
```

Comment:
Since this isn't a private project, it's probably not smart to store the dev credentials in source.  Ignore my comment above :)